Sources/StreamVideo/WebRTC/v2/PeerConnection/MediaAdapters/VideoCapturePolicy/VideoCapturePolicy.swift

```diff
@@ -1,143 +0,0 @@
-//
-// Copyright © 2025 Stream.io Inc. All rights reserved.
-//
-
-import CoreMedia
-import Foundation
-
-/// A class that defines video capture policies used by the `LocalVideoAdapter`
-/// to adjust video capture quality based on instructions from the SFU
-/// (Selective Forwarding Unit).
-///
-/// ## Overview
-/// On devices with limited processing power, video capture often starts at
-/// full quality, even if the SFU requests lower quality streams (e.g., half
-/// or quarter resolution). This mismatch can result in unnecessary processing
-/// overhead as the device continues to capture at a higher quality than needed.
-///
-/// By aligning the capture quality with the SFU's requests, `VideoCapturePolicy`
-/// helps reduce processing costs and improves performance, particularly on
-/// older or less powerful devices.
-public class VideoCapturePolicy: @unchecked Sendable {
```

Comment:
why is all this deleted?

Reply:
Because we don't rely any more on-device updates. We depend on the SFU to inform us when to change the capturing resolution.